src/utils/blockchainHelpers.js

```diff
@@ -38,6 +38,10 @@ export function getWeb3(cb) {
 	if (typeof web3 === 'undefined') {
     // no web3, use fallback
     console.error("Please use a web3 browser");
+    if (process.env.NODE_ENV === 'development') {
```

Comment:
extract to variable

Reply:
Do you mean the condition in the `if`? Something like:

```js
const devEnvironment = process.env.NODE_ENV === 'development'
if (devEnvironment) {
```

Reply:
yes exactly 👍